src/main/fc/runtime_config.c

```diff
@@ -15,10 +15,16 @@
  * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// FIXME this file has started to resemble a garbage bin.  move everything out into appropriate locations.
+
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "config/parameter_group.h"
+#include "config/parameter_group_ids.h"
```

Comment:
But for this PR this discussion is off topic, shall we open an issue to discuss this instead? The code coverage story got too big I think.